mpich-EL_7.spec

```diff
@@ -325,8 +324,6 @@ make testing
 %{_libdir}/%{name}/lib/pkgconfig/
 %{_libdir}/%{name}/lib/*.so
 %{_libdir}/%{name}/bin/mpicc
-%{_libdir}/%{name}/bin/mpic++
```

Comment:
if you re-enable the c++ bindings, do you need to remove these?

Reply:
Yeah.  b1c01c5.

So we are getting the full build, so all of that is working.  It's just working if and how `make testing` can be made to work.  It seems it was written to assume `make install` had already been run.  Which it has technically in the RPM build, but it was run with a `$DESTDIR`.